README.md

```diff
@@ -111,16 +118,26 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 ad5221e8ae73        ubuntu              "/bin/bash"         36 minutes ago      Up 36 minutes                     		suspicious_mirzakhani
 ```
 
-If this is your directory structure:
+Make sure that your `$SNAP_PATH` is set, if not:
 ```
-$GOPATH/src/github.com/intelsdi-x/snap/
-$GOPATH/src/github.com/intelsdi-x/snap-plugin-collector-docker/
+$ export SNAP_PATH=<snapDirectoryPath>/build
```

Comment:
That's exactly right @nanliu. We should work with the assumption that we teach people how to add `snapd` and `snapctl` to the path when they install it and that they have no concept of `$SNAP_PATH`.